src/common/sstream.cpp

```diff
@@ -128,6 +128,21 @@ size_t wxStringInputStream::OnSysRead(void *buffer, size_t size)
 // wxStringOutputStream implementation
 // ============================================================================
 
+wxStringOutputStream::wxStringOutputStream(wxString *pString, wxMBConv& conv)
+    : m_conv(conv)
+#if wxUSE_UNICODE
+    , m_unconv(0)
+#endif // wxUSE_UNICODE
+{
+    m_str = pString ? pString : &m_strInternal;
+
+#if wxUSE_UNICODE
+    m_pos = m_conv.FromWChar(NULL, 0, m_str->wc_str(), m_str->length());
```

Comment:
Wouldn't `m_pos = m_str->mb_str(conv).length();` be more efficient for the `wxUSE_UNICODE_UTF8` case (and a little bit simpler)?

Reply:
I think using `FromWChar(NULL)` is faster as it doesn't actually perform the conversion, but I've added an even faster path for the `wxUSE_UNICODE_UTF8` case now.